pages/common/ect.md

```diff
@@ -1,16 +1,25 @@
 # ect
 
-> Efficient Compression Tool (or ECT) is a C++ file optimizer. It supports PNG, JPEG, GZIP and ZIP files.
+> Efficient Compression Tool.
+> File optimizer written in C++. It supports `.png`, `.jpg`, `.gzip` and `.zip` files.
 > More information: <https://github.com/fhanau/Efficient-Compression-Tool>.
```

Comment:
Yes, you're right, the capital letters make it look slightly odd, but I'd like to still keep it the way it is, because the name of the GitHub repository has the capital letters in.